daprdocs/content/en/operations/configuration/invoke-allowlist.md

```diff
@@ -40,11 +40,11 @@ The following tables lists the different properties for access control, policies
 
 ### Operations
 
-| Property | Type   | Description |
-|----------|--------|-------------|
-| name     | string | Path name of the operations allowed on the called app. Wildcard "\*" can be used to under a path to match
-| httpVerb | list   | List specific http verbs that can be used by the calling app. Wildcard "\*" can be used to match any http verb. Unused for grpc invocation
-| action   | string | Access modifier. Accepted values "allow" (default) or "deny"
+| Property | Type   | Description                                                  |
+| -------- | ------ | ------------------------------------------------------------ |
+| name     | string | Path name of the operations allowed on the called app. Wildcard "\*" can be used to under a path to match. Wildcard "\**" can can be used to under multiple paths to match. |
+| httpVerb | list   | List specific http verbs that can be used by the calling app. Wildcard "\*" can be used to match any http verb. Unused for grpc invocation. |
```

Comment:
```suggestion
| httpVerb | list   | List specific http verbs that can be used by the calling app. Wildcard "\*" can be used to match any http verb. Unused for grpc invocation. |
```

Reply:
@CrazyHZM - Thanks for you contribution. Can you take a look at these suggested changes.